### PR TITLE
fix(Apple TV+): wrong show/movie name, not detecting playback

### PR DESCRIPTION
**Pull request by 7loud**
## Description 
* _Screenshot titles_
* Added setting to disable `smallImageKey`

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] I bumped the version of the presence in accordance with [Semver](https://devhints.io/semver)

## Screenshots
<details>
<summary>Wrong show/movie name</summary>

Screenshot_144
Screenshot_145
</details>

<details>
<summary>Not detecting playback</summary>

Before:
image

After:
Screenshot_146
</details>

### CODE REVIEW

File: websites/A/Apple TV+/presence.ts [hunk 1]
```diff
@@ -110,7 +123,7 @@ presence.on("UpdateData", async () => {
 						delete presenceData.endTimestamp;
 					}
 				} else {
-					presenceData.details = "Viewing show:";
+					presenceData.details = "Viewing Show:";
```
Comment (EncryptedDev) [inline websites/A/Apple TV+/presence.ts:126]: Show should be lowercase

File: websites/A/Apple TV+/presence.ts [hunk 2]
```diff
@@ -147,7 +160,7 @@ presence.on("UpdateData", async () => {
 						delete presenceData.endTimestamp;
 					}
 				} else {
-					presenceData.details = "Viewing movie:";
+					presenceData.details = "Viewing Movie:";
```
Comment (EncryptedDev) [inline websites/A/Apple TV+/presence.ts:163]: Here

File: websites/A/Apple TV+/presence.ts [hunk 3]
```diff
@@ -161,15 +174,15 @@ presence.on("UpdateData", async () => {
 		},
 		"/person/([a-zA-Z0-9-]+)": {
 			setPresenceData() {
-				presenceData.details = "Viewing person:";
+				presenceData.details = "Viewing Person:";
```
Comment (EncryptedDev) [inline websites/A/Apple TV+/presence.ts:177]: Ditto

File: websites/A/Apple TV+/presence.ts [hunk 4]
```diff
@@ -161,15 +174,15 @@ presence.on("UpdateData", async () => {
...
 				presenceData.state = document.querySelector(
 					"div.person-header__bio > h1"
 				)?.textContent;
 			}
 		},
 		"/settings": {
 			setPresenceData() {
-				presenceData.details = "Viewing their settings";
+				presenceData.details = "Viewing their Settings";
```
Comment (EncryptedDev) [inline websites/A/Apple TV+/presence.ts:185]: And here

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/A/Apple TV+/presence.ts [hunk 5]
```diff
@@ -184,11 +197,30 @@ presence.on("UpdateData", async () => {
...
+			data: ["smallImageKey"]
 		}
 	];
 
-	for (const [pathname, PData] of Object.entries(data.presence))
-		if (document.location.pathname.match(pathname)) PData.setPresenceData();
+	let presenceSelect;
+
+	for (const [pathname, PData] of Object.entries(data.presence)) {
+		if (document.location.pathname.match(pathname)) {
```
Comment (ImRodry) [inline websites/A/Apple TV+/presence.ts:211]:
```suggestion
		if (new RegExp(pathname).test(document.location.pathname)) {
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

**Review by ImRodry — CHANGES_REQUESTED** [review]
This presence is so weird it needs an entire overhaul but LGTM

**Review by ImRodry — APPROVED** [review]
Wrong button

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Thank you for your contribution! 🎉 @N0chteil

**Pull request merged**